spec/core_ext/process/pause_resume_spec.rb

```diff
@@ -1,15 +1,6 @@
-require 'sys-proctable'
-
 RSpec.describe Process do
-  let(:linux) { Gem::Platform.local.os == 'linux' }
-
   def get_process_status(pid)
-    process = Sys::ProcTable.ps(:pid => pid, :smaps => false, :cgroup => false)
-    linux ? process.state : process.status
-  end
-
-  def paused_status
-    linux ? 'T' : 4
+    `ps -p #{pid} -o stat=`.chomp.split('').first
```

Comment:
Since you are only grabbing the first character, instead if `.chomp.split('').first`, can we just use `[0]` ?

Reply:
Updated.